client/webserver/site/src/html/order.tmpl

```diff
@@ -0,0 +1,189 @@
+              <span class="match-data-label">Rate</span><br>
+              <span class="fs15">
+                {{$match.RateString}}
+              </span>
+            </div>
+            <div class="col-7 text-center">
+              <span class="match-data-label">Side</span><br>
+              <span class="fs15">{{$match.Side}}</span>
+            </div>
+            <div class="col-7 text-center">
+              <span class="match-data-label">Order Portion</span><br>
+              <span class="fs15">
+                {{$match.OrderPortion}}%
+              </span>
+            </div>
+          </div>
+
+          <div class="pt-3">
+            {{if len $match.Swap}}
+            <div class="px-3 pb-3">
+              <span class="match-data-label">Swap ({{$ord.FromSymbol}}, you)</span><br>
+              <a class="mono subtlelink" data-explorer-id="{{$ord.FromID}}"
+                data-explorer-coin="{{$match.SwapString}}">{{$match.SwapString}}</a>
+            </div>
+            {{end}}
+            {{if len $match.CounterSwap}}
+            <div class="px-3 pb-3">
+              <span class="match-data-label">Swap ({{$ord.ToSymbol}}, them)</span><br>
+              <a class="mono subtlelink" data-explorer-id="{{$ord.ToID}}"
+                data-explorer-coin="{{$match.CounterSwapString}}">{{$match.CounterSwapString}}</a>
+            </div>
+            {{end}}
+            {{if len $match.Redeem}}
+            <div class="px-3 pb-3">
+              <span class="match-data-label">Redemption ({{$ord.ToSymbol}}, you)</span><br>
+              <a class="mono subtlelink" data-explorer-id="{{$ord.ToID}}"
+                data-explorer-coin="{{$match.RedeemString}}">{{$match.RedeemString}}</a>
+            </div>
+            {{end}}
+            {{if len $match.CounterRedeem}}
+            <div class="px-3 pb-3">
+              <span class="match-data-label">Redemption ({{$ord.FromSymbol}}, them)</span><br>
+              <a class="mono subtlelink" data-explorer-id="{{$ord.FromID}}"
+                data-explorer-coin="{{$match.CounterRedeemString}}">{{$match.CounterRedeemString}}</a>
+            </div>
+            {{end}}
+            {{if len $match.Refund}}
+            <div class="px-3 pb-3">
+              <span class="match-data-label red">Refund ({{$ord.FromSymbol}}, you)</span><br>
+              <a class="mono subtlelink red" data-explorer-id="{{$ord.FromID}}"
+                data-explorer-coin="{{$match.RefundString}}">{{$match.RefundString}}</a>
+            </div>
+            {{end}}
+          </div>
+
+        </div>
+      {{end}}
+      </div>
+    {{end}}
+
+    {{- /* FUNDING COINS */ -}}
+    <div class="order-datum d-inline-block my-3">
+      <div class="text-left">FundingCoins</div>
+      <div class="fs14 text-left">
+        {{range $ord.FundingCoinIDs}}
+          <a class="subtlelink mono" data-explorer-id="{{$ord.FromID}}" data-explorer-coin="{{.}}">{{.}}</a><br>
+        {{end}}
```

Comment:
Clicking isn't opening a link for me.  It looks like a link, but clicking does nothing:

![image](https://user-images.githubusercontent.com/9373513/92494607-87569c00-f1bb-11ea-890f-945d9f2f3542.png)

`<a class="subtlelink mono" data-explorer-id="0" data-explorer-coin="b974f3a9e60a8ec01d0216e509b579004c51735f20e1783aaac7c7e802a4b113:0">b974f3a9e60a8ec01d0216e509b579004c51735f20e1783aaac7c7e802a4b113:0</a>`

BTC coins above, but same for DCR.  No messages in browser console.

Reply:
Also, `FundingCoins` could be split into two words or more.